src/components/Dashboard/DashboardComponent.jsx

```diff
@@ -149,4 +99,38 @@ class DashboardComponent extends Component {
     );
   }
 }
-export default withRouter(DashboardComponent);
+
+DashboardComponent.propTypes = {
+  lostAssets: PropTypes.object,
+  availableAssets: PropTypes.object,
+  damagedAssets: PropTypes.object,
+  allocatedAssets: PropTypes.object,
+  getAssetStatus: PropTypes.func.isRequired
+};
+
+DashboardComponent.defaultProps = {
+  lostAssets: {},
+  availableAssets: {},
+  damagedAssets: {},
+  allocatedAssets: {}
+};
+
+const mapStateToProps = ({ assetStatus }) => {
```

Comment:
Why haven't you separated `mapStateToProps` and the `connect` part to a container file? Remember that the component being connected to doesn't have to be in the same file as `mapStateToProps` and `mapDispatchToProps`. I'm suggesting the separation as this section should be part of the containers folder as explained [here](https://github.com/AndelaOSP/art-dashboard#containers-folder-_components) and it mostly entails copy pasting the logic.

You can also check [this redux example](https://redux.js.org/basics/exampletodolist#containers-filterlink-js)